src/main/java/com/st/utopia/agent/controller/Flights.java

```diff
@@ -0,0 +1,57 @@
+package com.st.utopia.agent.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.client.RestTemplate;
+
+import com.st.utopia.agent.model.Flight;
+
+public class Flights {
+	@Autowired
+	private RestTemplate restTemplate;
+
+	@Value("${utopia.search.API}")
+	private String searchAPI;
+	/**
+	 * Helper method to reduce the amount of repetitive code required for "get-all"
```

Comment:
Since this isn't only used for "get-all" methods, I suggest revising the Javadoc comment.